test/core-query-test.js

```diff
@@ -3,29 +3,37 @@
  * Modules dependencies
  */
 
-var mocha = require('mocha'),
+const mocha = require('mocha'),
 	figc = require('figc'),
 	assert = require('chai').assert,
 	libPath = process.env['SOLR_CLIENT_COV'] ? '../lib-cov' : '../lib',
 	solr = require( libPath + '/solr'),
 	sassert = require('./sassert');
 
 // Test suite
-var config = figc(__dirname + '/config.json');
-var client = solr.createClient(config.client);
-var basePath = [config.client.path, config.client.core].join('/').replace(/\/$/,"");
+const config = figc(__dirname + '/config.json');
+const client = solr.createClient(config.client);
+const basePath = [config.client.path, config.client.core].join('/').replace(/\/$/,"");
+const { createSchemaField } = require('./utils/schemaHelper')
 
 describe('Client#createQuery',function(){
 
+	before((cb) => {
```

Comment:
since we can't easily load `schema.xml` into Docker-based Solr, create necessary fields by hand.